tests/scm/scm-python-shared-atomspace.scm

```diff
@@ -19,9 +19,10 @@
 from opencog.atomspace import AtomSpace, TruthValue, types
 from opencog.type_constructors import atomspace
```

Comment:
If the atomspace is no longer global (below), then the import above is not needed. Same remark for `scm-python-arity.scm` test, above.